src/panels.cpp

```diff
@@ -1829,6 +1830,16 @@ const std::string panel_manager::get_current_layout_id() const
 void panel_manager::init()
 {
     load();
+    update_offsets( panel_manager::get_manager().get_current_layout().begin()->get_width() );
```

Comment:
This is happening inside of one of `panel_manager`'s functions already, so there's no need to request the singleton here. Also, `get_current_layout()` might [theoretically] provide no panels whatsoever: dereferencing `begin()` would yield UB in this case.

Reply:
Honestly I am terrible at comprehending OO stuff and I couldn't figure out how to make this work any other way.

Reply:
```suggestion
    update_offsets( get_current_layout().begin()->get_width() );
```

Reply:
I was under the impression this code path guaranteed a current layout was set

Reply:
Yeah it I looked more closely and that's definitely the case